test/media/time_ranges_utils_unit.js

```diff
@@ -5,10 +5,19 @@
  */
 
 goog.require('shaka.media.TimeRangesUtils');
+goog.require('shaka.util.Platform');
 
 describe('TimeRangesUtils', () => {
   const TimeRangesUtils = shaka.media.TimeRangesUtils;
 
+  let gapDetectionThreshold = 0.1;
```

Comment:
Since these tests don't need actual playback, you don't need platform-specific values.

Reply:
Done!